examples/config.ini

```diff
@@ -0,0 +1,4 @@
+[asterisk]
+host=127.0.0.1
+username=username
+secret=mysecret
```

Comment:
Not with a config file, however, I use events=off time to time, for daemons that launch commands only.